tools/trim_galore/trim_galore_wrapper.xml

```diff
@@ -248,97 +208,144 @@
             <when value="default" />
             <!-- Full/advanced params. -->
             <when value="custom">
-                <param name="rrbs" type="boolean" truevalue="--rrbs" falsevalue="" checked="True" label="Specifies that the input file was an MspI digested RRBS sample" />
-                <param name="non_directional" type="boolean" truevalue="--non_directional" falsevalue="" checked="False" label="Selecting this option for non-directional RRBS libraries" />
+                <param name="rrbs" type="boolean" truevalue="--rrbs" falsevalue="" checked="True"
+                    label="Specifies that the input file was an MspI digested RRBS sample" />
+                <param name="non_directional" type="boolean" truevalue="--non_directional" falsevalue="" checked="False"
+                    label="Selecting this option for non-directional RRBS libraries" />
             </when>  <!-- full -->
-      </conditional>  <!-- params -->
+        </conditional>  <!-- params -->
 
     </inputs>
     <outputs>
 
-        <data format="fastq" name="trimmed_reads_single" label="${tool.name} on ${on_string}: trimmed reads">
+        <data format="fastqsanger" name="trimmed_reads_single" from_work_dir="input_singles_trimmed.fq" label="${tool.name} on ${on_string}: trimmed reads">
           <filter>singlePaired['sPaired'] == "single"</filter>
-          <actions>
-                <action type="format">
-                  <option type="from_param" name="singlePaired.input_singles" param_attribute="ext" />
-                </action>
-          </actions>
         </data>
 
-        <data format="fastq" name="trimmed_reads_pair1" label="${tool.name} on ${on_string}: trimmed reads pair 1">
+        <collection name="trimmed_reads_paired_collection" type="paired" label="${tool.name} on ${on_string}: trimmed reads">
+            <filter>singlePaired['sPaired'] == "paired_collection"</filter>
+            <data name="forward" format="fastqsanger" from_work_dir="input_mate1_val_1.fq" />
+            <data name="reverse" format="fastqsanger" from_work_dir="input_mate2_val_2.fq" />
```

Comment:
It is possible to derive this from the inputs without requiring an action. Do you want to support multiple formats?